src/server/worker/driver/driver_unix.go

```diff
@@ -114,8 +114,10 @@ func (d *driver) linkData(inputs []*common.Input, dir string) error {
 		if input.Name == "" {
 			return errors.New("input does not have a name")
 		}
+
 		src := filepath.Join(dir, input.Name)
 		dst := filepath.Join(d.InputDir(), input.Name)
+
```

Comment:
I'd say remove these unless you feel they improve readability a lot. I generally feel like empty lines are only worth it for very distinct pieces of code. These ones seem related enough that the empty line doesn't add much and takes up more vertical space.